xapian-applications/omega/Makefile.am

```diff
@@ -98,7 +98,8 @@ TESTS =	atomparsetest$(EXEEXT)\
 	sdbntest$(EXEEXT)\
 	sourcetest\
 	urlenctest$(EXEEXT)\
-	utf8converttest$(EXEEXT)
+	utf8converttest$(EXEEXT)\
+	omindextest
```

Comment:
As a general principal, when adding a new entry to a list check if the list is currently in sorted order - if it is, you almost always want to add it so you preserve that sorted order...